scripts/config_template.yaml

```diff
@@ -9,11 +9,12 @@ data:
       indicator_type: 4-level vaccination and intent
       indicator: received a vaccination
       time_type: week
-    keep:
-      geography: geography
-      estimate: estimate
-      time_end: date
-    group_cols: [geography]
+
+# Columns to keep from each data set
+keep: ["geography", "estimate", "time_end"]
```

Comment:
```suggestion
keep: [geography, estimate, time_end]
```

Reply:
do we need to drop any of these other columns?

I'd suggest just letting them float along. We don't check for the *absence* of columns in any of our validations